packages/api-openfin/index.js

```diff
@@ -1,6 +1,11 @@
 window.Notification = function(title, options) {
```

Comment:
Actually, this would change the behaviour (slightly) and be inconsistent with Electron implementation - maybe best left as-is :+1: